src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/wiremock/IncidentsApiMockServer.kt

```diff
@@ -49,15 +52,30 @@ class IncidentsApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
-  fun stubIncidentForMigration(incidentId: Long = 1234) {
+  fun stubIncidentForMigration(incidentId: String = "4321") {
```

Comment:
incidentId now String (as mentioned above , and now defaulted to different value to the default nomisIncidentId)